helidon-ls-ext/helidon.jdt.ext/helidon.jdt.ext.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,27 @@
+Manifest-Version: 1.0
```

Comment:
Why is this checked-in ?

Reply:
This file is created by Tycho and is used during the project build and test